lxc/list.go

```diff
@@ -53,12 +55,24 @@ A single keyword like "web" which will list any instance with a name starting by
 A regular expression on the instance name. (e.g. .*web.*01$).
 A key/value pair referring to a configuration item. For those, the
 namespace can be abbreviated to the smallest unambiguous identifier.
+A key/value pair where the key is a shorthand. Multiple values must be delimited by ','. Available shorthands:
+  - type={'CONTAINER','VIRTUAL-MACHINE'}
+  - status={'RUNNING','STOPPED','FROZEN'}
+  - architecture={cpu-arch|string}
+  - device={device name|string}
+  - name={name|string}"
+  - location={location name|string}
+  - description={description|string}
```

Comment:
I think we should skip that one for now too. Descriptions are usually either missing (most common) or are set to something unique at which point you're better off filtering by name.